tests/conftest.py

```diff
@@ -180,9 +198,12 @@ def inner(
             assert_success: Optional[bool] = None,
             assert_failure: Optional[bool] = None,
             match_service_provider: Optional[ServiceProviderInstance] = None,
-            retry: Optional[bool] = False
+            retry: Optional[bool] = False,
+            is_this_your_device_screen: Optional[bool] = True,
+            select_this_is_my_device: Optional[bool] = False
```

Comment:
minor: if you end this line with a `,` it will

1. still be syntactically valid
2. save you the line noise next PR (the same noise you're getting with the `retry` arg)